pandas/tests/test_generic.py

```diff
@@ -1539,6 +1540,15 @@ def test_to_xarray(self):
                            expected,
                            check_index_type=False)
 
+    def test_deepcopy_empty(self):
```

Comment:
@bmcfee well you are changing fundamental things, which need full testing. 

``_construct`` is run with Series, DataFrame,Panel, to simply make a new one. Then you call a method/function and test if its a copy or not.